### ISSUE TITLE
Reflect is undefined on IE11 - Wrong vue-class-component version?

### ISSUE THREAD

**Issue by ThomasPoncet** [NightCatSama/vue-slider-component#320]
The package can't be loaded on IE11 (it uses Reflect that I don't polyfill in my project):

```
'Reflect' is undefined
```

It seems that the package uses an old version of vue-class-components (v6.3.0) and this version breaks on IE11.

A fix has been released on the vue-class-components ([v6.3.2](https://github.com/vuejs/vue-class-component/releases/tag/v6.3.2)).

If I manually clone the vue-slider-component GitHub repository, manually build the project and then replace in my project the directory `node_modules/vue-slider-component` by the directory where I cloned and build the vue-slider-component repository, I don't have the error on IE11 and vue-class-components is in version 6.3.2.

Is it possible to update vue-class-components on the published package? Or is the problem comming from my project?

Thank you for your help

**Comment by NightCatSama** [comment]
Thank you for your feedback, which will be updated later.

**Comment by NightCatSama** [comment]
I upgraded `vue-class-component` to `7.0.1` and re-released version `3.0.9`.

**Comment by ThomasPoncet** [comment]
@NightCatSama Thank your for your quick answer and action. I did try to update the package and it didn't fix the bug.

I did try something else: I did install vue-slider-component with npm in a new and clean directory

```sh
npm init
npm install -P vue-slider-component
```

And the version of vue-class-component in vue-slider-component seems still to be v6.3.0 (I open one of the js file in `node_modules/dist`, for example `node_modulesdist/vue-slider-component.umd.min.js` as it is the main, and I search `Reflect`).

I don't really understand how dependencies of vue-slider-component are packaged with vue-slider-component and then published on npm, but I feel like the wrong version of vue-class-component is sent to npm.

Let me know if I can add additional informations.
